index.js

```diff
@@ -3,7 +3,9 @@
  */
 
 import {AppRegistry} from 'react-native';
+import dotenv from 'dotenv;';
```

Comment:
import dotenv from 'dotenv;';

With React Native I got an error trying to run this with dotenv.

I looked into it and I believe you need to use [react-native-dotenv ](https://github.com/zetachang/react-native-dotenv ) or [react-native-config](https://github.com/luggit/react-native-config)

Reply:
Yes you are right and there was many other errors I forgot to test it thanks!